ionoscloud/resource_snapshot.go

```diff
@@ -187,48 +228,10 @@ func resourceSnapshotUpdate(ctx context.Context, d *schema.ResourceData, meta in
 func resourceSnapshotDelete(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	client := meta.(services.SdkBundle).CloudApiClient
 
-	rsp, apiResponse, err := client.SnapshotsApi.SnapshotsFindById(ctx, d.Id()).Execute()
```

Comment:
@cristiGuranIonos this function is significantly smaller now, we removed a lot of logic (I assume that the logic was there for a reason). What's the reasoning behind this?

Reply:
I see no reason for that logic there. I don't know why it was there originally, maybe API was less stable.